Tests/Segment-Tests/Support/TestUtilities.swift

```diff
@@ -107,3 +107,16 @@ class OutputReaderPlugin: Plugin {
         return event
     }
 }
+
+func waitUntilStarted(analytics: Analytics?) {
+    guard let analytics = analytics else { return }
+    var started = false
+    while started == false {
+        if let system: System = analytics.store.currentState() {
+            if system.started {
+                started = true
+            }
+        }
+        RunLoop.main.run(until: Date.distantPast)
```

Comment:
Would like a comment here saying you're ending the current runloop incase some newbs don't recognize it and take it out.

Reply:
nevermind, just noticed this is in tests